atlp-rwanda/script-heroes-frontend
# #174797466 Login with Facebook and Google

**What does this PR do?**
Login with Facebook and Google on Barefoot Nomad

**Description of Task to be completed?**
Add Facebook & Google Login button on Login Page

- Write code for Facebook & Google Login
- Create a fix PR on backend to redirect to Landing page after Success login

**How should this be manually tested?**
- Clone this repo git clone `https://github.com/atlp-rwanda/script-heroes-frontend`
- Checkout the branch git checkout `ft-login-with-facebook-and-google-#174797466`
- Install dependencies yarn install
- Run app yarn start
- Click on LOGIN WITH FACEBOOK or LOGIN WITH GOOGLE button
- Provide your Facebook or Google Credentials
- After a success login, you should be redirected to the Landing page

<img width="960" alt="Screenshot (193)" src="https://user-images.githubusercontent.com/49404485/108692843-0aaf4400-74f5-11eb-9fb6-4dbc76740b6d.png">

<img width="957" alt="Screenshot (195)" src="https://user-images.githubusercontent.com/49404485/108692486-91175600-74f4-11eb-855c-cf008cf6090a.png">

<img width="960" alt="Screenshot (194)" src="https://user-images.githubusercontent.com/49404485/108692431-80ff7680-74f4-11eb-81a9-242dbde00ec8.png">

**What are the relevant pivotal tracker stories?**
[#174797466](https://www.pivotaltracker.com/story/show/174797466)
